ckanext/canada/plugins.py

```diff
@@ -531,6 +533,10 @@ def get_validators(self):
                 validators.user_read_only,
             'user_read_only_json':
                 validators.user_read_only_json,
+            'no_future_date':
+                validators.no_future_date,
+            'canada_sort_prop_status':
+                validators.canada_sort_prop_status,
```

Comment:
these seem to be unrelated changes. You should always create branches from master, even if you intend to merge to staging, to avoid pulling in things like this.